src/RA_Dlg_GameLibrary.cpp

```diff
@@ -76,8 +76,7 @@ namespace ra {
 inline static void LogErrno() noexcept
 {
     std::array<char, 2048> buf{};
-    strerror_s(buf.data(), sizeof(buf), errno);
-    // TODO: Make StringPrintf support std::array
+    GSL_SUPPRESS_F6 Ensures(strerror_s(buf.data(), sizeof(buf), errno) == 0);
```

Comment:
Don't know if there's a C++ version of this function so it's still using the same one. This just needs to make sure the function only returns an `errno` of 0 (no error)